luaui/Widgets/camera_joystick.lua

```diff
@@ -313,7 +313,8 @@ local function SocketConnect(host, port)
 	client=socket.tcp()
 	client:settimeout(0)
 	res, err = client:connect(host, port)
-	if not res and not res=="timeout" then
+	if not res and err ~= "timeout" then
```

Comment:
`res` changed to `err`. Is this intended?

Reply:
yes, res in all those tests is incorrect, the error string is inside err